inert.js

```diff
@@ -110,6 +110,24 @@ class InertRoot {
    */
   _makeSubtreeUnfocusable(startNode) {
     composedTreeWalk(startNode, (node) => { this._visitNode(node); });
+
+    let activeElement = document.activeElement;
+    if (!document.contains(startNode)) {
```

Comment:
Yep it's precisely a walk up the ancestor chain (in Blink at least): https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/dom/Node.cpp?sq=package:chromium&dr=CSs&rcl=1471252828&l=766